metrics/interface/src/main/java/org/apache/iotdb/metrics/enums/MetricLevel.java

```diff
@@ -17,16 +17,22 @@
  * under the License.
  */
 
-package org.apache.iotdb.metrics.micrometer;
+package org.apache.iotdb.metrics.enums;
 
-import io.micrometer.core.instrument.Meter;
-import io.micrometer.core.instrument.Tags;
+public enum MetricLevel {
+  all(0),
+  normal(1),
+  important(2),
+  core(3);
 
-public class MeterIdUtils {
+  /** Level of metric */
+  int level;
 
-  private MeterIdUtils() {}
+  MetricLevel(int level) {
+    this.level = level;
+  }
 
-  public static Meter.Id fromMetricName(String metricName, Meter.Type type, String... tags) {
-    return new Meter.Id(metricName, Tags.of(tags), null, null, type);
+  public static boolean isHigher(MetricLevel level1, MetricLevel level2) {
```

Comment:
Method name is `isHigher`, but the logic is `higher or equal`

Reply:
yeap, I will fix this name.

Reply:
This has been finished